server/jetstream_benchmark_test.go

```diff
@@ -1174,3 +1174,184 @@ func BenchmarkJetStreamKV(b *testing.B) {
+					default:
+						rwRatioType = fmt.Sprintf("rwRatio=%0.1f", rwRatio)
+					}
+					for _, storageType := range benchmarkOptions.storageType {
+						bName := fmt.Sprintf(
+							"N=%d,R=%d,K=%d,storageType=%s,minObjSize=%db,maxObjSize=%db,workload=%s",
+							clusterSize,
+							replicas,
+							numKeys,
+							storageType.String(),
+							minObjSize,
+							maxObjSize,
+							rwRatioType,
+						)
+						b.Run(
+							bName,
+							func(b *testing.B) {
+								rng := rand.New(rand.NewSource(int64(seed)))
+
+								if verbose {
+									b.Logf("Running %s workload %s with %d messages", bName, bName, b.N)
+								}
+
+								if verbose {
+									b.Logf("Setting up %d nodes", replicas)
+								}
+								var (
+									connectURL string
+									cl         *cluster
+								)
+								if clusterSize == 1 {
+									s := RunBasicJetStreamServer(b)
+									defer s.Shutdown()
+									connectURL = s.ClientURL()
+								} else {
+									cl = createJetStreamClusterExplicit(b, "BENCH_OBJ_STORE", clusterSize)
+									defer cl.shutdown()
+									cl.waitOnClusterReadyWithNumPeers(replicas)
+									cl.waitOnLeader()
+									// connect to leader and not replicas
+									connectURL = cl.leader().ClientURL()
+								}
+								nc, js := jsClientConnectURL(b, connectURL)
+								defer nc.Close()
+
+								// Initialize object store
+								objStoreName := fmt.Sprintf("%sTEST", objStoreNamePrefix)
+								if verbose {
+									b.Logf("Creating ObjectStore %s with R=%d", objStoreName, replicas)
+								}
+								objStoreConfig := &nats.ObjectStoreConfig{
+									Bucket:   objStoreName,
+									Replicas: replicas,
+									Storage:  storageType,
+								}
+								objStore, err := js.CreateObjectStore(objStoreConfig)
+								if err != nil {
+									b.Fatalf("Error creating ObjectStore: %v", err)
+								}
+
+								// if cluster_size > 1, connect to stream leader
+								if cl != nil {
+									nc.Close()
+									connectURL = cl.streamLeader("$G", fmt.Sprintf("OBJ_%s", objStoreName)).ClientURL()
+									nc, js := jsClientConnectURL(b, connectURL)
+									defer nc.Close()
+									objStore, err = js.ObjectStore(objStoreName)
+									if err != nil {
+										b.Fatalf("Error binding to ObjectStore: %v", err)
+									}
+								}
+
+								// Initialize keys
+								if initKeys {
+									for n := 0; n < numKeys; n++ {
+										key := fmt.Sprintf("%s_%d", keyPrefix, n)
+										dataSz := rng.Intn(maxObjSize-minObjSize+1) + minObjSize
+										value := make([]byte, dataSz)
+										rng.Read(value)
+										_, err := objStore.PutBytes(key, value)
+										if err != nil {
+											b.Fatalf("Failed to initialize %s/%s: %v", objStoreName, key, err)
+										}
+									}
+								}
+
+								var (
+									errors float64
+									reads  float64
+									writes float64
+								)
+
+								// Each op is conducting a PUT/GET of avg(minBytes, maxBytes)
```

Comment:
> Each op is conducting a PUT/GET of avg(minBytes, maxBytes)

Not really.

Reply:
```suggestion
								// A single operation processes avg(minBytes, maxBytes)
```
How about this instead?